modules/ui_prefs.py

```
@@ -0,0 +1,207 @@
+
+        Falls back to defaults on any missing file, malformed JSON, or
+        malformed payload rather than raising to the UI. Unknown enum
+        strings degrade that field to ASK individually.
+
+        Returns:
+            The loaded (or default) preferences for every field.
+        """
+        defaults = {field: RememberDecision.ASK for field in PrefField}
+
+        try:
+            with open(self._prefs_path, 'r', encoding='utf-8') as f:
```

Ruff (0.16.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)